tests/backup/backup_test.go

```diff
@@ -5159,13 +5163,9 @@ var _ = Describe("{RestoreEncryptedAndNonEncryptedBackups}", func() {
 		for _, backupName := range backupNames {
 			backupUID := getBackupUID(backupName, orgID)
 			_, err = DeleteBackup(backupName, backupUID, orgID, ctx)
-			dash.VerifyFatal(err, nil, fmt.Sprintf("Deleting backup - [%s]", backupName))
+			dash.VerifyFatal(err, nil, "Deleting backup")
```

Comment:
We should be adding more useful info to the fail messages instead of deleting them, can you please add `backupName` and/or `backupUID`

Reply:
Added backup name